forc/src/utils/defaults.rs

```diff
@@ -5,10 +5,10 @@ pub(crate) fn default_manifest(project_name: &str) -> String {
 
     format!(
         r#"[project]
-name = "{}"
 author = "{}"
 entry = "main.sw"
 license = "Apache-2.0"
+name = "{}"
```

Comment:
When you change the order of `{}`, you have to also change what value they're assigned!

Reply:
Ah, I see, as in the name `{}` is not getting passed to the author's `{}`